### ISSUE TITLE
🚀 feat: figure with link: disable zoom

### ISSUE THREAD

**Issue by fuse314** [nunocoracao/blowfish#934]
**Describe the bug**
I have some "external links", I want to link to using the `figure` shortcode.
When I set the link using the `src` attribute, the link is clickable, but the cursor is displayed with the "zoom" icon and when I go "back" in the browser, the image is zoomed in.
I tried several approaches: set the href to the absolute url `https://...` and relative url `/...`, tried several tricks like setting the `class` attribute to "nozoom", with no luck.

**To Reproduce**
Use this code, i.E. on the root _index.md:
```
{{<figure href="the_link/" src="img/some_image.jpg" alt="test Figure with link." caption="this is a link: [click here](the_link/)" class="nozoom" >}}

{{<figure src="img/second_image.jpg" alt="this is a zoomable image." caption="this is a zoomable image" >}}
```
1. Hugo build
2. Open page in browser
3. Click on figure with link => the_link/ is opened
4. Click "back" in the browser
5. The image is displayed zoomed in.

**Expected behavior**
I see two possible solutions:
1. If a `figure` sets a link with the `src` attribute, the zoom functionality should be disabled.
2. Add a attribute like `nozoom=true` to suppress the zoom functionality on the image within the figure.

**Screenshots**
![figure_with_link](https://github.com/nunocoracao/blowfish/assets/927896/feb1de2d-7b07-4051-9d1c-42242ad272e2)

**Desktop (please complete the following information):**
 - OS: Windows 11, same behavior on Android.
 - Browser: Brave, Chrome, Edge: behavior like in screenshot. Firefox only shows the "zoom" mouse icon, the image does not stay zoomed in after browser "back" is pressed.
 - Version: latest

**Hugo & Blowfish versions**
Hugo 0.101.0, Blowfish latest from "main" branch. (6e0164d)

**Additional context**
If you are interested, I can create a pull request after deciding on the best solution.

Possibly connected to #679

**Comment by nunocoracao** [comment]
@fuse314 could you try passing a class "nozoom" to your figure - should work :)

**Comment by fuse314** [comment]
@nunocoracao see my code example and my text:
I already set the `class` to "nozoom".

The `class` is applied to the `figure` tag and not to the `img` tag within. The zoom javascript applies itself to all `img` tags that do not contain the "nozoom" class...

I already implemented the nozoom attribute in [my branch feature/figure_nozoom](https://github.com/fuse314/blowfish/tree/feature/figure_nozoom), it applies the css class directly to the `img` tag and works for me as expexted. Should I open a pull request?
